src/views/not-selected-signatory.njk

```diff
@@ -0,0 +1,20 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/breadcrumbs/macro.njk' import govukBreadcrumbs %}
+{% from 'govuk/components/button/macro.njk'      import govukButton %}
+
+{% block pageTitle %}
+  {{ Error }}
```

Comment:
What's this about? Can;t see this being passed in the view model. SHould the brackets be here?